var/spack/repos/builtin/packages/openspeedshop-utils/package.py

```diff
@@ -0,0 +1,343 @@
+
+            if spec.satisfies('+crayfe'):
+                # We need to build target/compute node
+                # components/libraries first then pass
+                # those libraries to the openspeedshop
+                # login node build
+                self.set_CrayLoginNode_cmakeOptions(spec, cmake_args)
+
+        return cmake_args
+
+    def set_defaultbase_cmakeOptions(self, spec, cmakeOptions):
+        # Appends to cmakeOptions the options that will enable
+        # the appropriate base level options to the openspeedshop
+        # cmake build.
+        python_exe = spec['python'].command.path
+        python_library = spec['python'].libs[0]
+        python_include = spec['python'].headers.directories[0]
+
+        BaseOptions = []
+
+        BaseOptions.append('-DBINUTILS_DIR=%s' % spec['binutils'].prefix)
+        BaseOptions.append('-DLIBELF_DIR=%s' % spec['elf'].prefix)
+        BaseOptions.append('-DLIBDWARF_DIR=%s' % spec['libdwarf'].prefix)
+        BaseOptions.append('-DPYTHON_EXECUTABLE=%s' % python_exe)
+        BaseOptions.append('-DPYTHON_INCLUDE_DIR=%s' % python_include)
+        BaseOptions.append('-DPYTHON_LIBRARY=%s' % python_library)
+        BaseOptions.append('-DBoost_NO_SYSTEM_PATHS=TRUE')
+        BaseOptions.append('-DBoost_NO_BOOST_CMAKE=TRUE')
+        BaseOptions.append('-DBOOST_ROOT=%s' % spec['boost'].prefix)
+        BaseOptions.append('-DBoost_DIR=%s' % spec['boost'].prefix)
+        BaseOptions.append('-DBOOST_LIBRARYDIR=%s' % spec['boost'].prefix.lib)
+        BaseOptions.append('-DDYNINST_DIR=%s' % spec['dyninst'].prefix)
+
+        cmakeOptions.extend(BaseOptions)
+
+    def set_mpi_cmakeOptions(self, spec, cmakeOptions):
+        # Appends to cmakeOptions the options that will enable
+        # the appropriate MPI implementations
+
+        MPIOptions = []
+
+        # openmpi
+        if spec.satisfies('+openmpi'):
+            MPIOptions.append('-DOPENMPI_DIR=%s' % spec['openmpi'].prefix)
+        # mpich
+        if spec.satisfies('+mpich'):
+            MPIOptions.append('-DMPICH_DIR=%s' % spec['mpich'].prefix)
+        # mpich2
+        if spec.satisfies('+mpich2'):
+            MPIOptions.append('-DMPICH2_DIR=%s' % spec['mpich2'].prefix)
+        # mvapich
+        if spec.satisfies('+mvapich'):
+            MPIOptions.append('-DMVAPICH_DIR=%s' % spec['mvapich'].prefix)
+        # mvapich2
+        if spec.satisfies('+mvapich2'):
+            MPIOptions.append('-DMVAPICH2_DIR=%s' % spec['mvapich2'].prefix)
+        # mpt
+        if spec.satisfies('+mpt'):
+            MPIOptions.append('-DMPT_DIR=%s' % spec['mpt'].prefix)
+
+        cmakeOptions.extend(MPIOptions)
+
+    def setup_environment(self, spack_env, run_env):
+        """Set up the compile and runtime environments for a package."""
+
+        # Common settings to both offline and cbtf versions
+        # of OpenSpeedShop
+        run_env.prepend_path('PATH', self.prefix.bin)
```

Comment:
This shouldn't be necessary, it is already present in Spack by default.

Reply:
Ok.  I will take that out.